modules/shortcodes/twitchtv.php

```diff
@@ -64,6 +64,9 @@ function wpcom_twitchtv_shortcode( $atts ) {
 		$url_args['channel'] = $user_id;
 	}
 
+	// See https://discuss.dev.twitch.tv/t/twitch-embedded-player-updates-in-2020/23956.
+	$url_args['parent'] = $_SERVER['HTTP_HOST'];
```

Comment:
Should we sanitize this before we use it in the URL, since `add_query_arg` doesn't do that below? Maybe it may be worth checking that it is set as well?

```suggestion
	$url_args['parent'] = isset( $_SERVER['HTTP_HOST'] ) ? rawurlencode( wp_unslash( $_SERVER['HTTP_HOST'] ) ) : '';
```

Reply:
I didn't escape it here since the URL is escaped below before being added to the `iframe` element. We should likely check that it's set though. I wish we had null coalesce :)

Reply:
I have just pushed a commit that addresses your feedback. Thank you!